heartbeat/apache

```diff
@@ -204,6 +204,15 @@ apache_start() {
 
 	validate_default_config || return $OCF_ERR_CONFIGURED
 
+	if [ -n "${APACHE_RUN_DIR}" ] && [ ! -d "${APACHE_RUN_DIR}" ]; then
+		mkdir -p "${APACHE_RUN_DIR}"
+	fi
+
+	if [ -n "${APACHE_LOCK_DIR}" ] && [ ! -d "${APACHE_LOCK_DIR}" ]; then
+		mkdir "${APACHE_LOCK_DIR}"
```

Comment:
Shouldn't this also be `mkdir -p`?

Reply:
Yeah. I've pushed a new update to fix it.